### PR TITLE
DOC: Remove inplace from user guide

### PR DESCRIPTION
**Pull request by phofl**
This starts with removing inplace from the user guide

### CODE REVIEW

**Comment by noatamir** [comment]
Just a thought: do we want to add any note/guidance somewhere for folks who might be looking specifically for inplace?

**Comment by noatamir** [comment]
Are these errors expected? (in: `advanced.rst`)
Screenshot of the amended example showing AttributeError

**Comment by phofl** [comment]
> Just a thought: do we want to add any note/guidance somewhere for folks who might be looking specifically for inplace?

Was thinking of writing up the PDEP and then linking the discussion in the whatsnew. There could be a better way though

Thx for catching, did not pay enough attention when modifying the examples. Fixed now

**Comment by lithomas1** [comment]
> > Just a thought: do we want to add any note/guidance somewhere for folks who might be looking specifically for inplace?
> 
> Was thinking of writing up the PDEP and then linking the discussion in the whatsnew. There could be a better way though
> 
> Thx for catching, did not pay enough attention when modifying the examples. Fixed now

@phofl Can you DM me on slack? I was also thinking about writing up a PDEP (didn't get too far though).

File: doc/source/getting_started/comparison/includes/copies.rst [hunk 1]
```diff
@@ -13,11 +13,3 @@ or overwrite the original one:
       df = df.sort_values("col1")
 
 .. note::
-
-   You will see an ``inplace=True`` keyword argument available for some methods:
```
Comment (lithomas1) [inline doc/source/getting_started/comparison/includes/copies.rst:17]: Should be fine to keep this one since we're discouraging inplace usage here?

Comment (phofl) [reply doc/source/getting_started/comparison/includes/copies.rst:17]: But with the wrong arguments, I'd rather add it again after finishing the pdep

**Comment by phofl** [comment]
Everyone ok with going ahead and merging?

**Review by lithomas1 — APPROVED** [review]

**Review by mroeschke — APPROVED** [review]

**Comment by mroeschke** [comment]
Thanks @phofl

**Pull request merged**